backend/functions/lib/configs/application-configuration.ts

```diff
@@ -30,40 +27,48 @@ export class ApplicationConfiguration implements AppConfig {
   }
 
   private getDbConfig(database: string): IDbConfig {
-    const dbConfig: IDbConfig = {
-      server: process.env.MSSQL_HOST,
-      database: database,
-      user: process.env.MSSQL_USER,
-      password: '',
-      azureManagedIdentity: process.env.AZURE_MANAGED_IDENTITY || '',
-      authentication: {
-        type: 'azure-active-directory-msi-app-service',
-      },
-      pool: {
-        max: 10,
-        min: 0,
-        idleTimeoutMillis: 30000,
-      },
-      options: {
-        encrypt: Boolean(process.env.MSSQL_ENCRYPT),
-        trustServerCertificate: Boolean(process.env.MSSQL_TRUST_UNSIGNED_CERT),
-      },
+    const server = process.env.MSSQL_HOST;
+    const port: number = Number(process.env.MSSQL_PORT) || 1433;
+    const encrypt: boolean = Boolean(process.env.MSSQL_ENCRYPT);
+    const trustServerCertificate: boolean = Boolean(process.env.MSSQL_TRUST_UNSIGNED_CERT);
+    const type = process.env.MSSQL_AUTH_TYPE || 'azure-active-directory-default';
+    const user = process.env.MSSQL_USER;
+    const password = process.env.MSSQL_PASS;
+    const clientId = process.env.MSSQL_CLIENT_ID; // User Identity client
```

Comment:
I would prefer to use a more meaningful variable name than have to include the comment describing what it is.

Reply:
As Fritz noted, `type` is probably also not a good choice for variable name because that is a reserved word in TypeScript. So maybe something like `authType` and `userIdentityClientId`.